tests/test_decimal.py

```diff
@@ -0,0 +1,28 @@
+"""
```

Comment:
I know the test files could do with cleaning up, but this doesn't require a standalone test file.

Please move into `test_json`.

Reply:
Have done that now :)